### DISCUSSION TITLE
How can I lie to clients about the world age?

### DISCUSSION THREAD

**Discussion by ryleu** [Minestom/Minestom#1108] [Help] [answered]
I am writing a toggleable shader that depends on the world age. How can I lie to clients about how old the instance is? I'm not able to se the `worldAge` and I can't stop the `TimeUpdate` packet, as far as I can tell. What should I do?

Thanks in advance.

**Comment by emortaldev** [comment, marked as answer]
Does sending another TimeUpdate packet not work?

**Reply by ryleu** [reply]
It does not. The client just ignores the second packet, and the server sends a new packet every second or so, so even if it did work, it would produce flashing on the screen.

**Reply by ryleu** [reply]
Nevermind, that does work, it was my shader that was broken. Thanks!
